src/sentry/sentry_metrics/transactions.py

```diff
@@ -0,0 +1,13 @@
+"""Base module for transaction-related metrics"""
+from enum import Enum
+
+
+class TransactionMetricsKey(Enum):
+    """
+    Identifier for a transaction-related metric.
+
+    The values are the metric names sumbitted by Relay.
+    """
+
+    TRANSACTION = "transaction"
+    TRANSACTION_STATUS = "transaction.status"
```

Comment:
I like the enum, but `transaction.status` is a tag and not a metric. So maybe make a separate `TransactionTagKey` enum?

Reply:
+1 on this. If you choose to create an Enum for the `transaction.status`, I would also create an enum in a follow up PR for `session.status` as well for consistency